scripts/generate-change-list.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const { execSync } = require('child_process');
+
+const wrapText = require('wrap-text');
+
+/**
+ * Return a `Date` indicating when a Git tag was created.
+ */
+function getTagDate(tag) {
+  const result = execSync(`git tag --list "${tag}" "--format=%(taggerdate)"`, {
+    encoding: 'utf-8',
+  });
+  return new Date(result.trim());
+}
+
+/**
+ * Return the name of the most recently created Git tag.
+ */
+function getLastTag() {
+  const result = execSync('git tag --list --sort=-taggerdate', {
+    encoding: 'utf-8',
+  });
+  const tags = result.split('\n').map(line => line.trim());
+
+  if (tags.length === 0) {
+    return null;
+  }
+  return tags[0];
+}
+
+/**
+ * Iterate over pages of items in a GitHub API response and yield each item.
+ */
+async function* iterateItems(octokit, response) {
```

Comment:
This is an async generator, ie. a generator function which always yields promises. Such a thing is convenient for paging through API responses.

Reply:
The function name `iterateItems` is rather generic; I can see that it is meant to be reused across different GH API responses...wondering if there is a slightly more specific naming that could be used here.